### PR TITLE
Add flag for test output 

### PR DESCRIPTION
**Pull request by ankit-privado**
Add flag to export a separate json file which contain only unfiltered flow when --test-output or -tout flag provided.
JIRA Ticket: https://privado-ai.atlassian.net/browse/LE-82

### CODE REVIEW

**Comment by ankit-privado** [comment]
@pandurangpatil @dbMundada @ojaswa1942 PTAL.

**Review by abhstabs — APPROVED** [review]
LGTM!

File: cmd/scan.go [hunk 1]
```diff
@@ -71,6 +71,7 @@ func defineScanFlags(cmd *cobra.Command) {
 	scanCmd.Flags().Bool("disable-flow-separation-by-data-element", false, "Experimental: If specified, filtering of flow using 'flow separation by data element algorithm' will be avoided")
 	scanCmd.Flags().Bool("disable-2nd-level-closure", false, "Experimental: If specified, 2nd level source derivation will be turned on")
 	scanCmd.Flags().Bool("generate-unresolved-name-report", false, "Flag to enable generation unresolved method name reports")
+	scanCmd.Flags().Bool("test-output", false, "Flag to generate unfiltered flow output getting direclty from joern")
```
Comment (ojaswa1942) [inline cmd/scan.go:74]: Please modify the description to:
```
If specified, additionally generates an unfiltered flow report
```

Also, let's rename the flag to `generate-unfiltered-report`.

File: cmd/scan.go [hunk 2]
```diff
@@ -89,6 +90,7 @@ func scan(cmd *cobra.Command, args []string) {
 	disableFlowSeperationByDataElement, _ := cmd.Flags().GetBool("disable-flow-separation-by-data-element")
 	disable2ndLevelClosure, _ := cmd.Flags().GetBool("disable-2nd-level-closure")
 	generateUnresolvedNameReport, _ := cmd.Flags().GetBool("generate-unresolved-name-report")
+	testOutput, _ := cmd.Flags().GetBool("test-output")
```
Comment (ojaswa1942) [inline cmd/scan.go:93]: Rename the variable in accordance with the new flag name.

**Review by ojaswa1942 — CHANGES_REQUESTED** [review]

File: cmd/scan.go (hunk 1 above)
Comment (ankit-privado) [reply cmd/scan.go]: Done, changed to `generate-unfiltered-report` PTAL @ojaswa1942

File: cmd/scan.go (hunk 2 above)
Comment (ankit-privado) [reply cmd/scan.go]: Done @ojaswa1942

**Comment by ankit-privado** [comment]
@ojaswa1942 done with all change.

**Pull request merged**